pkg/kv/kvserver/replica_app_batch.go

```diff
@@ -562,6 +562,12 @@ func (b *replicaAppBatch) ApplyToStateMachine(ctx context.Context) error {
 	if err := b.batch.Commit(sync); err != nil {
 		return errors.Wrapf(err, "unable to commit Raft entry batch")
 	}
+	// TODO(sumeer): Plumb BatchCommitStats into appBatchStats. appBatchStats
```

Comment:
Wouldn't it be appropriate to sum up the `BatchCommitStats` just like the other fields of `appBatchStats`: https://github.com/cockroachdb/cockroach/blob/fa17185faa53b535b18e377945275e791ba17253/pkg/kv/kvserver/app_batch.go#L44-L49

raft handling applies at most ~64mb of entries, which will be split across a few batches. I think everything here already almost works? Just need to aggregate the stats (snippet above) and also print it here https://github.com/cockroachdb/cockroach/blob/b29901bb6bcc62d29d9c2a08e6361043c49962c5/pkg/kv/kvserver/replica_raft.go#L629

I might be missing something.